tls/s2n_client_extensions.c

```diff
@@ -69,7 +69,10 @@ int s2n_client_extensions_send(struct s2n_connection *conn, struct s2n_stuffer *
         total_size += (sizeof(s2n_preferred_hashes) * num_signature_algs * 2) + 6;
     }
 
-    uint16_t application_protocols_len = conn->config->application_protocols.size;
+    struct s2n_blob *app_protocols;
```

Comment:
Readability nit: Can we rename `app_protocols` to `client_app_protocols`?

Reply:
Done.